apps/explorer/tests/test_views.py

```diff
@@ -1014,6 +1014,92 @@ def test_renders_message_on_success(self):
         )
 
 
+class PixelSetUnselectViewTestCase(CoreFixturesTestCase):
```

Comment:
I think we say "deselect", not "unselect" https://english.stackexchange.com/questions/18465/unselect-or-deselect